crates/viewer/re_view/src/query.rs

```diff
@@ -66,29 +65,29 @@ pub fn range_with_blueprint_resolved_data<'a, 'b>(
 /// [`crate::HybridResults`].
 ///
 /// If `query_shadowed_components` is true, store components will be queried, even if they are not used.
-pub fn latest_at_with_blueprint_resolved_data<'a, 'b>(
+// TODO(#10460): Should take identifiers instead of descriptors.
+pub fn latest_at_with_blueprint_resolved_data<'a>(
```

Comment:
Same here: A leftover, but the `component_types` in this function's comment is outdated.